Pre-release update for v22.0.0 (#1320)
TomographicImaging/CIL: NOTICE.txt

CCPi Core Imaging Library (CIL).

> Copyright 2017-2022 Rutherford Appleton Laboratory STFC

Reviewer: ```suggestion
Copyright 2017-2022 United Kingdom Research and Innovation
```

Note in the CIL Astra files, which we updated after discussing with the lawyers, we have:
https://github.com/TomographicImaging/CIL/blob/00b55e453fb6e8f81be296f09bd059133f46c885/Wrappers/Python/cil/plugins/astra/operators/AstraProjector2D.py#L2-L3

I also wonder if next to our names it should be (STFC, UKRI)

Copyright 2017-2022 University of Manchester

This software product is developed for the Collaborative Computational 

> Project in Tomographic Imaging CCPi (http://www.ccpi.ac.uk/) at RAL STFC (http://www.stfc.ac.uk), University of Manchester

Reviewer: I think this should be changed to mention UKRI
Other files say:

> This work is part of the Core Imaging Library (CIL) developed by CCPi 
> (Collaborative Computational Project in Tomographic Imaging), with 
> substantial contributions by UKRI-STFC and University of Manchester.

Reviewer: (see suggested changes below)

> This software product is developed for the Collaborative Computational 
> Project in Tomographic Imaging CCPi (http://www.ccpi.ac.uk/) at RAL STFC (http://www.stfc.ac.uk), University of Manchester

Reviewer: ```suggestion
This software product is developed for the Collaborative Computational 
Project in Tomographic Imaging CCPi (http://www.ccpi.ac.uk/) with
substantial contributions by UKRI-STFC and The University of Manchester.
```
